### DISCUSSION TITLE
copy versus __deepcopy__?

### DISCUSSION THREAD

**Discussion by tgkolda** [sandialabs/pyttb#381] [General]
I understand that we have the `copy` method to create a deep copy. I'm confused, however, about the `__deepcopy__` method. Why is that there?

**Comment by ntjohnson1** [comment]
`__deepcopy__` is a python construct to support copy operations https://docs.python.org/3/library/copy.html

**Comment by tgkolda** [comment]
Got it. I'll add this clarification to the documentation.
